distro/configuration/ampathforms/tb_case_enrollment_v1.0.json

```diff
@@ -906,6 +1171,26 @@
   ],
   "processor": "EncounterFormProcessor",
   "encounterType": "9a199b59-b185-485b-b9b3-a9754e65ae57",
+  "postSubmissionActions": [
+    {
+      "actionId": "ProgramEnrollmentSubmissionAction",
+      "enabled":"tbProgramType === '160541AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA'",
+      "config": {
+      "enrollmentDate": "tbRegDate",
+      "programUuid": "58005eb2-4560-4ada-b7bb-67a5cffa0a27",
+      "completionDate": "outcomeTBRx"
+    }
+  },
+  {
+    "actionId": "ProgramEnrollmentSubmissionAction",
+    "enabled":"tbProgramType === '160052AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA'",
+    "config": {
+    "enrollmentDate": "tbRegDate",
+    "programUuid": "00f37871-0578-4ebc-af1d-e4b3ce75310d",
+    "completionDate": "outcomeTBRx"
+  }
+}
+  ],
```

Comment:
This indentation is off and this is pending a PR to the form engine so am not sure why this was merged to dev in the first place.